src/olympia/addons/indexers.py

```diff
@@ -176,6 +188,12 @@ def extract_document(cls, obj):
                 'min': min_, 'min_human': min_human,
                 'max': max_, 'max_human': max_human,
             }
+        data['author'] = [{'name': a.name, 'id': a.id, 'username': a.username}
```

Comment:
mmm I replied to this but it's not showing, maybe I forgot to hit the button. Anyway yes, `listed_authors` is better, I'll change it.